draft-ietf-httpbis-message-signatures.md

```diff
@@ -196,32 +199,78 @@ The following table shows example canonicalized values for header fields, given
 
 |Header Field|Canonicalized Value|
 |--- |--- |
-|`(cache-control)`|max-age=60, must-revalidate|
-|`(date)`|Tue, 07 Jun 2014 20:51:35 GMT|
-|`(server)`|www.example.com|
-|`(x-empty-header)`||
-|`(x-obs-fold-header)`|Obsolete line folding.|
-|`(x-ows-header)`|Leading and trailing whitespace.|
+|`cache-control`|max-age=60, must-revalidate|
+|`date`|Tue, 07 Jun 2014 20:51:35 GMT|
+|`server`|www.example.com|
+|`x-empty-header`||
+|`x-obs-fold-header`|Obsolete line folding.|
+|`x-ows-header`|Leading and trailing whitespace.|
 {: title="Non-normative examples of header field canonicalization."}
 
+## Dictionary Structured Field Members
```

Comment:
this PR does not specify the actual format of a dictionary

Reply:
disregard - wasn't aware of raft-kamp-httpbis-structure-01

Reply:
FYI, current WG version of the Structured Fields draft is here: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure